modules/develop/pages/data-transforms/how-transforms-work.adoc

```diff
@@ -14,7 +14,7 @@ Data transforms let you run common data streaming tasks, like filtering, scrubbi
 
 Data transforms use https://webassembly.org/[WebAssembly^] (Wasm) engines inside a Redpanda broker, allowing Redpanda to control the entire transform lifecycle. For example, Redpanda can stop and start transforms when partitions are moved or to free up system resources for other tasks.
 
-Data transforms take data from an input topic and map it to an output topic. For each topic partition, a leader is responsible for handling the data. Redpanda runs a virtual machine (VM) on the same CPU core (shard) as these partition leaders to execute the transforms function.
+Data transforms take data from an input topic and map it to one or more output topics. For each topic partition, a leader is responsible for handling the data. Redpanda runs a Web Assembly virtual machine (VM) on the same CPU core (shard) as these partition leaders to execute the transforms function.
```

Comment:
Looks like you are picking up changes to the 24.1 docs. Should this branch be based off that instead of main?